tests/zenko_e2e/util/azure.py

```diff
@@ -67,6 +68,7 @@ def delete_blob(self, name):
 def cleanup_azure_bucket(bucket, delete_bucket=True):
     blobs = list(bucket.objects.all())
     for blob in blobs:
-        bucket.delete_blob(blob.name)
+        if blob.name.startswith(conf.OBJ_PREFIX):
```

Comment:
Does this account for the bucket_match false addition?

Reply:
I.e. is the source bucket name prepended to the key name on the destination?

Reply:
At the moment none of the python e2e tests run with bucket_match=false, and I think at this point other changes would have to be made to support it